test/node/limit-download-upload.js

```diff
@@ -0,0 +1,82 @@
+const fixtures = require('webtorrent-fixtures')
```

Comment:
Could you run `lebab` to modernize this code? See https://github.com/webtorrent/webtorrent/pull/2134/files